synapse/rest/media/v1/media_repository.py

```diff
@@ -158,6 +158,16 @@ def create_content(self, media_type, upload_name, content, content_length,
     @defer.inlineCallbacks
     def get_local_media(self, request, media_id, name):
         """Responds to reqests for local media, if exists, or returns 404.
+
+        Args:
+            request(twisted.web.http.Request)
+            media_id (str)
```

Comment:
sorry, but please could you doc media_id, here and all the other places it's used? it would help me be less confused about the difference between it and file_id.